evap/evaluation/migrations/0158_examtype.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 5.2 on 2025-10-06 18:57
+
+import django.contrib.postgres.fields
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def _migrate(apps, _schema_editor):
+    Evaluation = apps.get_model("evaluation", "Evaluation")
+    ExamType = apps.get_model("evaluation", "ExamType")
+    exam_type = ExamType.objects.create(name_de="Klausur", name_en="Exam")
+    Evaluation.objects.filter(models.Q(name_de="Klausur") | models.Q(name_en="Exam")).update(exam_type=exam_type)
```

Comment:
Please do this for a few evaluations in `test_data.json`. Currently, every evaluation has exam type null.

Reply:
Ok, then I just create same random exams and dump the test data afterwards?